dae-alright/src/Redux/reducers/simpleReducer.js

```diff
@@ -0,0 +1,11 @@
+import SIMPLE_ACTION from "../constants/action-type"
+
+export default (state = {}, action) => {
+    if (action.type === SIMPLE_ACTION) {
```

Comment:
use a `switch case` here instead. Also, create a variable called `const initialState = {}` and assign it to the state rather than `{}` to make future modifications `easy peezy lemon squizy`

Reply:
Okay